tests/test_downloader_handlers.py

```diff
@@ -809,7 +821,13 @@ def _mocked_date(self, date):
 
     def test_extra_kw(self):
         try:
-            S3DownloadHandler.from_crawler(get_crawler(), extra_kw=True)
+            crawler = get_crawler()
+            create_instance(
+                objcls=S3DownloadHandler,
+                settings=crawler.settings,
```

Comment:
this argument can be skipped, because when crawler is passed, settings are default to crawler.settings in create_instance